cg/models/orders/samples.py

```diff
@@ -195,6 +195,10 @@ def str_to_int(cls, v: str) -> int | None:
 class PacBioSample(OrderInSample):
     _suitable_project = OrderType.PACBIO_LONG_READ
 
+    capture_kit: str | None
```

Comment:
If we want this as optional we need to set a default
```suggestion
    capture_kit: str | None = None
```